src/services/config.js

```diff
@@ -119,19 +129,21 @@ class ConfigData {
     }
   }
 
-  addProfile(id, accountSid, region) {
-    // Clean all the inputs.
+  addProfile(id, accountSid, region, apiKey, apiSecret) {
```

Comment:
Looks like addProfile was previously used only to add an entry into projects and credentials were stored in another call, but since you're doing both here, you might need to validate the key and secret inputs

Reply:
We're generating these apiKey and apiSecrets at runtime through an API call at runtime : [Link](https://github.com/twilio/twilio-cli/blob/fe17479e3b50033894d194247961083802ec403f/src/commands/profiles/create.js#L238) and shouldn't require any further validations. 

Also, just checked we seem to be saving without any further validations in keytar currently as well.